keel-titus-plugin/keel-titus-plugin.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+  `java-library`
+  id("kotlin-spring")
+}
+
```

Comment:
I probably don't need all these dependencies, but I'll tighten them up as I build more of this.